tasks/service-clickhouse.yml

```diff
@@ -1,13 +1,9 @@
 ---
 
-- name: CLICKHOUSE | Copy systemd unit configuration file
-  template:
-    src: "{{ clickhouse_service_template_path }}"
-    dest: "/etc/systemd/system/{{ clickhouse_service }}.service"
-    mode: 0644
-    owner: root
-    group: root
-  notify: reload-clickhouse
+- name: CLICKHOUSE | Remove file (delete file if exists)
```

Comment:
If we update the service template with the new settings this change isn't necessary